Status.sh

```diff
@@ -152,6 +149,16 @@ then
   freedns_id_pass="\Zb\Z5FreeDNS ID and pass\Zn"
 fi
 
+# Mark existence of problem characters in API_SECRET
+apisec_literal=$(grep 'API_SECRET=' /etc/nsconfig | sed 's/^.*=//')
```

Comment:
this 3 lines try to parse api_secret but I'm not sure they will succeed.
Can you do something like:
. /etc/nsconfig
and then you should have API_SECRET as bash has interpreted it.

Reply:
That's what we used to have.  But, it will only work if the content of the file respects matched quotation marks.  If someone includes a single quote in their password, that options will fail.
That's the reason I changes the process to literally break down the content and treat it as a bunch of characters.

Reply:
I have tested this with all possible combinations I could think of.
If you give me something you think may fail, I will show you what it does.

For example, tell me what password you think will fail.  For example, you can say try the following.

ThisIsAPasswor'd

And I will show you what the code does.

Reply:
As discussed offline, the original version is probably ok.